examples/accesscontextmanager/accesslevelcondition.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: accesscontextmanager.gcp.upbound.io/v1beta1
+kind: AccessLevelCondition
+metadata:
+  annotations:
+    meta.upbound.io/example-id: accesscontextmanager/v1beta1/accesslevelcondition
+    upjet.upbound.io/manual-intervention: "Requires organization level IAM permissions"
+  labels:
+    testing.upbound.io/example-name: access-level-conditions
+  name: access-level-conditions
+spec:
+  forProvider:
+    accessLevelSelector:
+      matchLabels:
+        testing.upbound.io/example-name: access-level-service-account
+    devicePolicy:
+    - osConstraints:
+      - osType: DESKTOP_CHROME_OS
+      requireAdminApproval: false
+      requireCorpOwned: true
+      requireScreenLock: false
+    ipSubnetworks:
+    - 192.0.4.0/24
+    members:
+    - user:test@google.com
+    - user:test2@google.com
+    - serviceAccount:${google_service_account.created-later.email}
+    negate: false
+    regions:
+    - IT
+    - US
+
+---
+
+apiVersion: accesscontextmanager.gcp.upbound.io/v1beta1
+kind: AccessLevel
+metadata:
+  annotations:
+    meta.upbound.io/example-id: accesscontextmanager/v1beta1/accesslevelcondition
+    upjet.upbound.io/manual-intervention: "Requires organization level IAM permissions"
+  labels:
+    testing.upbound.io/example-name: access-level-service-account
+  name: access-level-service-account
+spec:
+  forProvider:
+    basic:
+    - conditions:
+      - devicePolicy:
+        - osConstraints:
+          - osType: DESKTOP_CHROME_OS
+          requireScreenLock: true
+        regions:
+        - CH
+        - IT
+        - US
+    name: accessPolicies/${google_access_context_manager_access_policy.access-policy.name}/accessLevels/chromeos_no_lock
+    parentSelector:
+      matchLabels:
+        testing.upbound.io/example-name: access-policy
```

Comment:
I'm not sure if this example works with this parameter as in the `example/accesscontextmanager/accesslevel.yaml` example: https://github.com/upbound/provider-gcp/pull/331#discussion_r1257120924